components/curriculum/sidebar.css

```diff
@@ -1,206 +1,114 @@
-.curriculum-content-container .sidebar {
+.curriculum-content-container .left-sidebar {
```

Comment:
Not necessarily for before release, but these styles are incredibly close to the left-sidebar component styles: I think we should add an issue to merge them